pkg/authorization/apis/authorization/conversion.go

```diff
@@ -241,3 +243,38 @@ func Convert_rbac_PolicyRules_To_authorization_PolicyRules(in []rbac.PolicyRule)
 	}
 	return rules
 }
+
+func translate_Origin_Annotations_To_rbac_Annotation(role *rbac.ClusterRole) {
+	// can't use const in pkg/oc/admin/policy because of import cycle
+	_, ok := role.Annotations["openshift.io/reconcile-protect"]
+	if ok {
```

Comment:
Something like this should work:

```go
func convert_authorization_annotation_To_rbac_annotation(in map[string]string) map[string]string {
	// out bootstrap objects should have the default annotation so ideally we should not hit this
	if in == nil {
		return nil
	}

	// can't use const in pkg/oc/admin/policy because of import cycle
	if in["openshift.io/reconcile-protect"] == "true" {
		// make a deep copy so we can mutate
		out = ... // deep copy annotation
		out["rbac.authorization.kubernetes.io/autoupdate"] = "false"
		delete(out, "openshift.io/reconcile-protect")
		return out
	}

	return in
}
```

```go
out.ObjectMeta = convert_authorization_annotation_To_rbac_annotation(in.ObjectMeta)
```